platform/android/java/app/config.gradle

```diff
@@ -1,13 +1,13 @@
 ext.versions = [
-    androidGradlePlugin: '8.2.0',
+    androidGradlePlugin: '8.5.0',
     compileSdk         : 34,
     // Also update 'platform/android/export/export_plugin.cpp#OPENGL_MIN_SDK_VERSION'
-    minSdk             : 21,
+    minSdk             : 24,
```

Comment:
Depending on when this is merged, we may have to find a way to make it optional, as this will increase the minimum Android version requirement to 7.0. [Support may be good enough](https://apilevels.com/) by then.